### PR TITLE
Feature: Add option to select dark/light/automatic theme

### PR DESCRIPTION
**Pull request by wutschel**
## Description

This PR adds a new setting option "Theme" which allows to set `UIUserInterfaceStyle` for the whole app. Possible settings are:
- Automatic (use the system mode, default)
- Dark (use Dark Mode)
- Light (use Light Mode)

As this only works on devices running iOS 13 or higher the setting itself is named "Theme (iOS13+)".

Screenshots: https://abload.de/img/bildschirmfoto2022-107afrd.png

## Summary for release notes

Feature: Add option to select dark/light/automatic theme

### CODE REVIEW

**Comment by kambala-decapitator** [comment]
should it go into the hotfix version?

**Comment by wutschel** [comment]
I would keep it out. This setting would otherwise keep showing with the "iOS13+" comment for users which stick to an older OS. You agree?

**Comment by kambala-decapitator** [comment]
yes, I agree

> I would keep it out. This setting would otherwise keep showing with the "iOS13+" comment for users which stick to an older OS. You agree?

but we'd still support iOS 12 ;)

**Comment by wutschel** [comment]
At least we don't bother the users with iOS 9, 10 and 11 with this.

**Comment by kambala-decapitator** [comment]
currently there's no upcoming requirement to switch to Xcode 14, I'd continue supporting iOS 9+. What do you think?

**Comment by wutschel** [comment]
> currently there's no upcoming requirement to switch to Xcode 14, I'd continue supporting iOS 9+. What do you think?

The NowPlaying changes I did on top of the Xcode 14 migration I can most likely decouple. Leaves us with Xcode 14 fixing some layout problems with iPhone 14+ "island". Not more than that.

File: XBMC Remote/Settings.bundle/Root.plist [hunk 1]
```diff
@@ -102,7 +102,29 @@
...
-			<string>Filemode changes needs app restart</string>
+			<string>Filemode and Theme changes need app restart.</string>
+		</dict>
+		<dict>
+			<key>DefaultValue</key>
+			<string>automatic</string>
+			<key>Key</key>
+			<string>theme_mode</string>
+			<key>Title</key>
+			<string>Theme (iOS13+)</string>
```
Comment (kambala-decapitator) [inline XBMC Remote/Settings.bundle/Root.plist:113]:
```suggestion
			<string>Theme (iOS 13+)</string>
```

File: XBMC Remote/Settings.bundle/en.lproj/Root.strings [hunk 2]
```diff
@@ -25,7 +25,8 @@
 "Enable contents cache" = "Enable contents cache";
 "Override Lockscreen" = "Override Lockscreen";
 "Show all files in filemode" = "Show all files in filemode";
-"Filemode changes needs app restart" = "Filemode changes needs app restart";
+"Theme (iOS13+)" = "Theme (iOS13+)";
```
Comment (kambala-decapitator) [inline XBMC Remote/Settings.bundle/en.lproj/Root.strings:28]:
```suggestion
"Theme (iOS 13+)" = "Theme (iOS 13+)";
```

**Review by kambala-decapitator — APPROVED** [review]
I was referring to the previous discussion in the PR. But after checking the changes I see it's not a problem :)

File: XBMC Remote/Settings.bundle/Root.plist (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/Settings.bundle/Root.plist]: Fine for me, done.

File: XBMC Remote/Settings.bundle/en.lproj/Root.strings (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/Settings.bundle/en.lproj/Root.strings]: Fine for me, done.

**Pull request merged**